astropy/stats/histogram.py

```diff
@@ -206,11 +205,9 @@ def freedman_bin_width(data, return_bins=False):
     dx = 2 * (v75 - v25) / (n ** (1 / 3))
 
     if return_bins:
-        dmin, dmax = data.min(), data.max()
-        if dx < 1e-6:
-            dx = median_absolute_deviation(data)
         if dx < 1e-6:
```

Comment:
This is very strange: `dx` has units of the data, so why would one use an absolute number? Something like
```
if dx < 1e-6 * (v75 + v25):
```
would make more sense (which quite likely a different pre-factor).

Reply:
Thanks for the review!